scripts/utils/clone-repos.sh

```diff
@@ -0,0 +1,124 @@
+#!/usr/bin/env bash
+
+# variables
+declare GROUP_NAME="<source-group-name>"
+declare SOURCE_PAT="<source-pat>"
+declare SOURCE_FQDN="<source-fqdn>"
+declare TARGET_PAT="<target-pat>"
+declare TARGET_FQDN="<target-fqdn>"
+
+declare DEBUG_FLAG=false
+
+# includes
+source ../lib/shell_logger.sh
+source ../lib/sh_arg.sh
```

Comment:
It doesn't look like we are using sh_arg.sh. Unless we opt to use command line params, we can probably remove this. (see above comment)

Reply:
Used